src/domWrapper.ts

```diff
@@ -137,4 +140,21 @@ export class DOMWrapper<ElementType extends Element>
 
     return new DOMWrapper(parentElement).trigger('change')
   }
+
+  async trigger(eventString: DomEventName | string, options?: TriggerOptions) {
```

Comment:
I still think we need overrides to make the inference better:
```
async trigger(eventString: DomEventName, options?: TriggerOptions)
async trigger(eventString: string, options?: TriggerOptions)
```

Reply:
Hmm, didn't realise it would make it better but it did.